cmd/channel/email/main.go

```diff
@@ -38,17 +51,50 @@ func (ch *Email) SendNotification(req *plugin.NotificationRequest) error {
 	}
 
 	var msg bytes.Buffer
-	_, _ = fmt.Fprintf(&msg, "To: %s\n", strings.Join(to, ","))
-	_, _ = fmt.Fprintf(&msg, "Subject: [#%d] %s %s is %s\n\n", req.Incident.Id, req.Event.Type, req.Object.Name, req.Incident.Severity)
-
 	plugin.FormatMessage(&msg, req)
 
-	err := smtp.SendMail(ch.GetServer(), nil, ch.From, to, bytes.ReplaceAll(msg.Bytes(), []byte("\n"), []byte("\r\n")))
+	return enmime.Builder().
+		ToAddrs(to).
+		From(ch.SenderName, ch.SenderMail).
+		Subject(fmt.Sprintf("[#%d] %s %s is %s", req.Incident.Id, req.Event.Type, req.Object.Name, req.Incident.Severity)).
+		Header("Message-Id", fmt.Sprintf("<%s-%s>", uuid.New().String(), ch.SenderMail)).
+		Text(msg.Bytes()).
+		Send(ch)
+}
+
+func (ch *Email) Send(reversePath string, recipients []string, msg []byte) error {
+	var (
+		client *smtp.Client
+		err    error
+	)
+
+	if ch.Encryption == EncryptionTLS {
+		client, err = smtp.DialTLS(ch.GetServer(), nil)
+	} else {
+		client, err = smtp.Dial(ch.GetServer())
+	}
 	if err != nil {
 		return err
 	}
+	defer client.Close()
```

Comment:
`Close()` returns an error. If we are going to ignore `defer` errors, maybe make it explicit.
```suggestion
	defer func() { _ = client.Close() }()
```